README.md

```diff
@@ -1 +1,19 @@
-# Flog 0.79
+# Flog 0.79: Flask Blog
+
+## ffmpeg is required! [https://ffmpeg.org/]
+Intallation of ffmpeg on UBUNTU:
+##
+- sudo add-apt-repository ppa:mc3man/trusty-media
+- sudo apt-get update
+- sudo apt-get dist-upgrade
+- sudo apt-get install ffmpeg
+
+Installation of ffmpeg on Windows:
+- http://ffmpeg.zeranoe.com/builds/
+- download and unpack the static build
+- Add [path]\ffmpeg\bin to system path
+
+##
+1. python setup.py install
+2. export FLASK_APP = run.py
```

Comment:
haha)) u know, i was very ̶l̶a̶z̶y̶ busy) and just copy-paste this from ur readme)))